lambdas/tabular_preview/Dockerfile

```diff
@@ -0,0 +1,67 @@
+ARG FUNCTION_DIR="/function"
+
+FROM debian:trixie-20251020@sha256:72547dd722cd005a8c2aa2079af9ca0ee93aad8e589689135feaed60b0a8c08d AS base
```

Comment:
you should use AWS Lambda image (i.e. public.ecr.aws/lambda/python:3.12.XXX) as a base like here

https://github.com/quiltdata/quilt/blob/d5fa977a888f793b0493894a86ef2e61f20fc9da/lambdas/indexer/Dockerfile#L3

unless there are strong reasons to not do that